WEEK-3 FUNCTIONS/7-Melike/alfabetical_order.py

```diff
@@ -0,0 +1,8 @@
+###################### 3- alfabetical_order #################
+
+
+def alfabetical_order(x):
+    list1= []
+    list1 = x.split("-")
+    list1.sort()
+    print("-".join(list1))
```

Comment:
Herhangi bir giriş yapılmadığı için bir çıktı alınmıyor. Kullanıcıdan kelime girişi yapmasını istemelisiniz.